go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/crossplane/crossplane
 
 go 1.20
 
+replace github.com/crossplane/crossplane-runtime => github.com/pedjak/crossplane-runtime v0.0.0-20231004110250-0b3151613115
```

Comment:
this PR depends on https://github.com/crossplane/crossplane-runtime/pull/552 and it needs to merged first

Reply:
why does it?

Reply:
ack

Reply:
Runtime PR merged.

Reply:
updated to latest crossplane